.env.sample

```diff
@@ -30,8 +30,12 @@ KAFKA_GROUP_ID = "projects"
 SUBMISSION_TOPIC = "dev.sl.projects.submissions"                                // Kafka topic name for pushing projects submissions
 PROJECT_SUBMISSION_TOPIC = "dev.sl.projects.submissions"                        // project submission topic 
 
-# SUNBIRD LOCATION AND USER READ
-USER_SERVICE_URL = "http://user-service:3000"                                   // service used for user profile read location search are using this base url
+# USER SERVICE URL
```

Comment:
@Prajwal17Tunerlabs remove ml service urls from env sample and remove all unwanted fields

Reply:
removed unwanted services from env sample